### PR TITLE
chore: Add docs for data explorer v2

### PR DESCRIPTION
**Pull request by connoratrug**
#### Checklist
- [ ] Functionality works & meets specifications
- [ ] Code reviewed
- [ ] Code unit/integration/system tested
- [ ] Migration step added in case of breaking change
- [ ] User documentation updated
- [ ] (If you have changed REST API interface) view-swagger.ftl updated
- [ ] Test plan template updated
- [ ] Clean commits
- [ ] Added Feature/Fix to release notes

### CODE REVIEW

**Comment by connoratrug** [comment]
[Add doc that explain (custom)card vs table view, setting options and other de2 actions](https://trello.com/c/sqdFrOlk/421-add-doc-that-explain-customcard-vs-table-view-setting-options-and-other-de2-actions)

File: docs/guide-explore.md [hunk 1]
```diff
@@ -257,3 +257,89 @@ You now get a table representing the amount of samples in both biobanks per type
...
+The data explorer v2 can be added to the (main)menu via a redirect plugin 
+- Open the menu-manager
+- In de section 'Create a new menu item'
+  - Select ```redirect``` as plugin type
+  - Choose a name ( for example: 'Dataexplorer')
+  - Paste a link to app as the queryString
+```url=%2F@molgenis-ui%2Fdata-explorer%2Fdist%2Findex.html```
+  - Add the item to the menu 
+	
+### Data explore v2 features
```
Comment (MaxPostema) [inline docs/guide-explore.md:277]: missing 'r' in data explorer

File: docs/guide-explore.md [hunk 2]
```diff
@@ -257,3 +257,89 @@ You now get a table representing the amount of samples in both biobanks per type
...
+
+Differences with data explorer v2 will be listed 
+
+#### Table actions are accessible using the toolbar
+(hover over the buttons to view action labels).
+- Add button: The add button allows the user to add a new row to the table ( only visible if user has 'ADD_DATA' rights).
+- Search input:  Allows the user to search the entire table.
+- Orderby dropdown: Allows user to oder the data by a given field ( clicking the button reverses the order)
```
Comment (MaxPostema) [inline docs/guide-explore.md:285]: In DE2 its listed as "Order by"

File: docs/guide-explore.md [hunk 3]
```diff
@@ -257,3 +257,89 @@ You now get a table representing the amount of samples in both biobanks per type
 Finally, select "patientid" in the third dropdown, the distinct attribute.
 The table will update to show you how many patients with at least one sample of a specific type are available in a biobank.
 
+## Data Explorer v2 (Experimental)
+
+The data explorer can be used to explorer the data
```
[the remark this answers is not in the document]
Comment (connoratrug) [reply docs/guide-explore.md:262]: i prefer the original;
'Displaying data is also a lot more customizable in v2.' uses the v1 vs v2 , i would like just describe v2 ( so we can remove v1 at some point and keep de v2 doc). A phrase like '..also a lot more' does not feel right for a doc imo. the purpose of doc i think should be to describe the de-2 ( as apposed to introduce the v2 and offset it against the v1), more something to mention in a demo or a slidedeck )

File: docs/guide-explore.md (hunk 1 above)
Comment (connoratrug) [reply docs/guide-explore.md:277]: tnks , please feel free to fix typo's inline during the review

File: docs/guide-explore.md [hunk 4]
```diff
@@ -257,3 +257,89 @@ You now get a table representing the amount of samples in both biobanks per type
...
+- View selector: Toggle between card or grid view
+- Settings: (only visible as admin) opens up table settings form
+
+#### Card view and custom cards
+Data explorer v2 has the ability to show data as a grid of cards or as a data table ( similar to data explorer v1)
+
+
+##### Card view with default card
+By default each card shows a list show as a list of label / value pairs. Each item in the list is a column in the data table.
+The number of items show can be configured using the table settings
```
Comment (MaxPostema) [inline docs/guide-explore.md:299]: Missing . at end of line.

File: docs/guide-explore.md (hunk 2 above)
Comment (connoratrug) [reply docs/guide-explore.md]: add space

File: docs/guide-explore.md [hunk 5]
```diff
@@ -257,3 +257,89 @@ You now get a table representing the amount of samples in both biobanks per type
...
+Differences with data explorer v2 will be listed 
+
+#### Table actions are accessible using the toolbar
+(hover over the buttons to view action labels).
+- Add button: The add button allows the user to add a new row to the table ( only visible if user has 'ADD_DATA' rights).
+- Search input:  Allows the user to search the entire table.
+- Orderby dropdown: Allows user to oder the data by a given field ( clicking the button reverses the order)
+  - for quick access clicking on the column header labels sort the table by that column.
+  - a second click reverses the sort order
+- Show/Hide columns: Can be used to hide ( and unhide) columns ( or card fields)
```
[the remark this answers is not in the document]
Comment (connoratrug) [reply docs/guide-explore.md]: updated the ones i could find

File: docs/guide-explore.md [hunk 6]
```diff
@@ -257,3 +257,89 @@ You now get a table representing the amount of samples in both biobanks per type
...
+- Download button: Downloads the table as an exel file
+- View selector: Toggle between card or grid view
+- Settings: (only visible as admin) opens up table settings form
+
+#### Card view and custom cards
+Data explorer v2 has the ability to show data as a grid of cards or as a data table ( similar to data explorer v1)
```
[the remark this answers is not in the document]
Comment (connoratrug) [reply docs/guide-explore.md:292]: its mentioned a few lines below

File: docs/guide-explore.md (hunk 4 above)
Comment (connoratrug) [reply docs/guide-explore.md]: added the missing dots

File: docs/guide-explore.md [hunk 7]
```diff
@@ -257,3 +257,89 @@ You now get a table representing the amount of samples in both biobanks per type
...
+<p>by {{record.author.label }}</p>
+```
+
+Customised card:
+
+![Dataexplorer default_card_example](./images/dataexplorer/de-2-docs-custom-card-book.png)
+
+
+A template can contain any valid vue template code, this includes any valid html ( 
+including inline styles or scripts).
```
[the remark this answers is not in the document]
Comment (connoratrug) [reply docs/guide-explore.md:325]: not to my knowledge, will try to find out , do you know where this is mentioned ?

[the remark this answers is not in the document]
Comment (MaxPostema) [reply docs/guide-explore.md:325]: Note sure, some dark corner of my mind. May be bogus.

[the remark this answers is not in the document]
Comment (connoratrug) [reply docs/guide-explore.md]: i removed the part about the script , the focus should be on vue template code ( including html)

**Pull request merged**